ppr-ui/src/composables/useTableFeatures.ts

```diff
@@ -0,0 +1,25 @@
+/** Common Table Functions. **/
+export const useTableFeatures = () => {
```

Comment:
Good idea to put table functions in a separate composable. It could be extended as we go. 👍🏻